File: python/ebeats.py
#!/usr/bin/env python

from datetime import datetime

def ebeats():
    t = datetime.now()
    perHour = 1000.0 / 24.0
    perMinute = 1000.0 / (24.0 * 60.0)
    perSecond = 1000.0 / (24.0 * 60.0 * 60.0)
    beats = t.hour * perHour + t.minute * perMinute + t.second * perSecond
    return "%0.2f" % beats

File: csharp/ebeats.cs
// C# implemenation of ebeats
//
// mcs ebeats.cs && mono ebeats.exe

using System;
public class Ebeats {
	static public void Main() {
		DateTime dt = DateTime.UtcNow;
		Console.WriteLine( String.Format("@{0:0.00}", 
				(dt.Hour * 3600 + 
				 dt.Minute * 60 + 
				 dt.Second)  * 0.01157407));
	}
}
		



